text/0006-architecture-review-process.md

```diff
@@ -23,15 +23,15 @@ For both a Roadmap Review and a Design Review the **timetable** is as follows:
 
 1. The team should set the stage:
     * Summarize in one or two sentences the proposal to be reviewed. This should be agreed between the team asking for review and the reviewer.
-    * Find a **reviewer**:
+    * Find a **reviewer** (See “Outputs and Audience” below for the rationale):
       - For a Roadmap Review this will be the least senior manager with the ability to address the problem in its entirety.
       - For a Design Review this will be an experienced engineer in the problem domain outside of the team willing to put time into the problem. As engineers don’t review their own code they shouldn’t review their own designs. The principle is to get the most informed and least biased feedback possible.
     * Optionally, for larger reviews, find someone to **chair** the review. This can facilitate the process (i.e. scheduling the meeting, managing the clock, ensuring minutes are taken, etc.) and enables the reviewer to concentrate on the review itself. The rest of this document uses 'chair' for the administrative role. For smaller reviews the reviewer also does the tasks of the chair.
 
-2. The team produces a **Review Packet** designed to document the proposal identified in step 1. The production of the Review Packet is expected to be collaborative and iterative to ensure that the review focuses on the key issues so the team presents its best case. This is particularly true of a Roadmap Review where the consequences of the review are likely to be more significant. The Review Packet includes:
-    * A lay summary of the problem space which is focused on defining a shared language and identifying the key forces behind the problem
+2. The team produces a **Review Packet** designed to document the proposal identified in step 1. The Review Packet includes:
+    * A lay summary of the problem space which is focused on defining a shared language and identifying the key forces behind the problem.
```

Comment:
Since you're changing this line anyway, here's a recommendation to simplify it (for clarity):

* A lay summary of the problem space that defines a shared language and identifies the key forces behind the problem.

Reply:
Fixed in: 219e8bf4487d965dd8b7a961c3c9487db2b06b54